crates/cxx-qt-lib/src/core/qbytearray.cpp

```diff
@@ -92,6 +92,12 @@ qbytearrayAppend(QByteArray& byteArray, ::std::uint8_t ch)
   byteArray.append(static_cast<char>(ch));
 }
 
+void
+qbytearrayPrepend(QByteArray& byteArray, ::std::uint8_t ch)
```

Comment:
keep the order the same as the header, so this will move to after Len and before Remove